libs/designsystem/src/lib/components/charts/chart-config/chart-stock-config/chart-stock-config.ts

```diff
@@ -0,0 +1,198 @@
+  private static plugins = [annotationPlugin, Filler, Tooltip];
+  private static registeredElements = [
+    ...this.scales,
+    ...this.controllers,
+    ...this.elements,
+    ...this.plugins,
+  ];
+
+  private static pluginsRegistered = false;
+
+  public static registerPlugins() {
+    if (StockChartConfig.pluginsRegistered) {
+      return;
+    }
+
+    this.registeredElements.forEach((scale) => {
+      Chart.register(scale);
+    });
+    StockChartConfig.pluginsRegistered = true;
+  }
+
+  public static basicConfig: ChartConfiguration<ChartType> = {
+    type: 'line',
+    data: {
+      datasets: [],
+    },
+    options: {
+      responsive: true,
+      animation: {
+        duration: 0,
+      },
+      layout: {
+        padding: {
+          left: 20,
+          right: 20,
+          top: 30,
+          bottom: 0,
+        },
+      },
+      backgroundColor: getThemeColorRgbString('semi-light', 0.5),
+      scales: {
+        x: {
+          adapters: {
+            date: {
+              locale: StockChartConfig.STOCK_CHART_LOCALE_DEFAULT,
+            },
+          },
+          grid: {
+            lineWidth: 0,
+          },
+          ticks: {
+            maxRotation: 0,
+            autoSkipPadding: 120,
+            font: {
+              size: parseInt(fontSize('xs')),
+            },
+            callback(tickValue) {
+              const dateInMiliseconds = parseInt(this.getLabelForValue(parseInt(tickValue + '')));
+
+              return getChartStockShortDate(
+                dateInMiliseconds,
+                StockChartConfig.STOCK_CHART_LOCALE_DEFAULT
+              );
+            },
+          },
+        },
+        y: {
+          position: 'right',
+          display: true,
+          grid: {
+            drawBorder: false,
+          },
+          ticks: {
+            display: true,
+            font: {
+              size: parseInt(fontSize('xs')),
+            },
+          },
+        },
+      },
+      elements: {
+        line: {
+          tension: 0, // Smooth curve (0 = no smoothing)
+          borderWidth: 2,
+        },
+        point: {
+          hitRadius: 20,
+          radius: 0,
+          hoverRadius: 0,
+          hoverBorderWidth: 0,
+        },
+      },
+      normalized: true,
+      plugins: {},
+    },
+  };
+
+  public static tooltipPlugin: Partial<TooltipOptions<'line'>> = {
+    padding: 10,
+    enabled: true,
+    mode: 'index',
+    intersect: false,
+    backgroundColor: getThemeColorHexString('semi-light'),
+    titleColor: getThemeColorHexString('semi-light-contrast'),
+    bodyColor: getThemeColorHexString('semi-light-contrast'),
+    caretSize: 0,
+    bodySpacing: 5,
+    titleSpacing: 5,
+    borderColor: 'transparent',
+    callbacks: {
+      title: (toolTipItem) => {
+        const dateInMiliseconds = parseInt(toolTipItem[0].label); // TODO: Right now we use index === 0. I expect multiple datasets will change this
```

Comment:
Recently we have tried to have less TODO's in code. Can we move it to an issue, or add it to an existing issue?